internal/util/tree/tree.go

```diff
@@ -61,69 +63,98 @@ var (
 	cyan   = color.New(color.FgCyan)
 )
 
+// CreateObjectTree creates a new tablewriter.Table for the object tree.
+// Returns a new tablewriter.Table for the object tree.
+func CreateObjectTree(w io.Writer) *tablewriter.Table {
+	cfg := tablewriter.Config{
+		Row: tw.CellConfig{
+			Formatting: tw.CellFormatting{AutoWrap: tw.WrapNone},
+			Alignment:  tw.CellAlignment{Global: tw.AlignLeft},
+			Padding:    tw.CellPadding{Global: tw.Padding{Left: "", Right: "  "}},
+		},
+		Header: tw.CellConfig{
+			Alignment: tw.CellAlignment{Global: tw.AlignLeft},
+		},
```

Comment:
From a quick test, if we set 
```suggestion
			Formatting: tw.CellFormatting{AutoWrap: tw.WrapNone},
			Alignment:  tw.CellAlignment{Global: tw.AlignLeft},
			Padding:    tw.CellPadding{Global: tw.Padding{Left: "", Right: " "}},
```
We get a more consistent alignment between headers and rows, wdyt?
(same for CreateObjectTreeV1Beta1)